### DISCUSSION TITLE
添加的列表项不能删除(Added list items cannot be deleted)

### DISCUSSION THREAD

**Discussion by cutelolly** [community/community#50546] [Mobile]
对于用户自身添加的列表项，后续不能编辑也不能删除。
List items added by the user themselves cannot be edited or deleted later.

**Comment by aconsuegra** [comment]
Hi @cutelolly, thanks fir getting in touch! What do you mean exactly with "list items"? Do you mean repositories added to user lists?

**Comment by jsoref** [comment]
Could you possibly include some screen shots?

**Comment by Appfanatic456** [comment]
I think I saw this on https://www.red-c.co.uk/ somewhere, theres a blog about it but I could not find it, maybe you'll have better look. but I think you have to click on the icon, and the edit button is hidden, you may have to hold option if your on a mac while going to the bar at the top and then the remove button should appear. But before you do that, provide some screenshots
